src/submission/submission-entities.ts

```diff
@@ -88,6 +88,11 @@ export interface CreateRegistrationRecord {
   readonly sampleType: string;
 }
 
+export interface CommitClinicalSubmissionCommand {
```

Comment:
This could actually be used for validate and approve, we could name this something more general?